tester_app/app/views/hello_world/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Client Rendering</h1>
+<%= react_component("HelloWorldApp", {initialName: "Stranger"}, prerender: false, trace: true) %>
```

Comment:
Did this pass linting?

I think the ruby linter will check for spaces around braces.

and pass in @hello_world_props

and have `generator_function: false be explicit`